packages/usage-reporting-protobuf/package.json

```diff
@@ -5,19 +5,24 @@
   "type": "module",
   "exports": {
     ".": {
-      "import": "./generated/protobuf.mjs",
-      "require": "./generated/protobuf.cjs",
-      "types": "./generated/protobuf.d.ts"
+      "import": "./generated/esm/protobuf.js",
+      "require": "./generated/cjs/protobuf.js",
+      "types": {
+        "require": "./generated/cjs/protobuf.d.ts",
+        "default": "./generated/esm/protobuf.d.ts"
+      }
     }
   },
-  "main": "generated/protobuf.cjs",
-  "module": "generated/protobuf.mjs",
-  "types": "generated/protobuf.d.ts",
+  "main": "generated/cjs/protobuf.js",
+  "module": "generated/esm/protobuf.js",
+  "types": "generated/esm/protobuf.d.ts",
   "scripts": {
-    "generate": "rm -rf generated && mkdir generated && npm run pbjs-cjs && npm run pbjs-esm && npm run pbts",
-    "pbjs-cjs": "apollo-pbjs --target static-module --out generated/protobuf.cjs --wrap commonjs --force-number --no-from-object src/reports.proto",
-    "pbjs-esm": "apollo-pbjs --target static-module --out generated/protobuf.mjs --es6 --force-number --no-from-object src/reports.proto",
-    "pbts": "mv generated/protobuf.{c,}js && apollo-pbts -o generated/protobuf.d.ts generated/protobuf.js && mv generated/protobuf.{,c}js",
+    "generate": "rm -rf generated && mkdir generated && mkdir generated/esm && mkdir generated/cjs && npm run pbjs-cjs && npm run pbjs-esm && npm run pbts",
+    "pbjs-cjs": "apollo-pbjs --target static-module --out generated/cjs/protobuf.cjs --wrap commonjs --force-number --no-from-object src/reports.proto",
+    "pbjs-esm": "apollo-pbjs --target static-module --out generated/esm/protobuf.mjs --es6 --force-number --no-from-object src/reports.proto",
+    "pbts-cjs": "mv generated/cjs/protobuf.{c,}js && apollo-pbts -o generated/cjs/protobuf.d.ts generated/cjs/protobuf.js && echo {\"module\": \"commonjs\"} > generated/cjs/package.json",
+    "pbts-esm": "mv generated/esm/protobuf.{m,}js && apollo-pbts -o generated/esm/protobuf.d.ts generated/esm/protobuf.js",
```

Comment:
Do we need a `package.json` in the esm dir as well?

Reply:
If not, can you explain why not? If yes, can you augment your smoke test to catch this issue?

Reply:
Not needed because it'll use the "module" from this package.json
The reason it's needed in cjs is to override the... wait a sec I messed that up it should be `{"type":"commonjs"}` not `{"module": "commonjs"}` 
interesting that node somehow understood my intent.

Anyway the goal is to tell node to use `{"type":"commonjs"}` when resolving files in `generated/cjs`
As for files in `generated/esm` node will go up to this package.json file and see `{"type":"module"}`

Reply:
Investigating…

Reply:
Alright I figured it out.
The "right" way is to output `{"type":"commonjs"}` to show the intent.
However, it was working with `{"module": "commonjs"}` because of how node resolves the package.json file.
Simply having an empty `{}` package.json is actually enough to make it work because of this 

https://nodejs.org/api/packages.html#type
```
If the nearest parent package.json lacks a "type" field, or contains "type": "commonjs", .js files are treated as CommonJS.
If the volume root is reached and no package.json is found, .js files are treated as CommonJS.
```

Reply:
Ah, that explains it perfectly! Thanks for digging. So the existence of the `package.json` at all implied commonjs. This would have only broken in the absence of package file altogether.